src/action.js

```diff
@@ -29,7 +29,28 @@ async function exportSecrets() {
 
     const defaultOptions = {
         prefixUrl: vaultUrl,
-        headers: {}
+        headers: {},
+        https: {}
+    }
+
+    const tlsSkipVerify = core.getInput('tlsSkipVerify', { required: false }) != 'false';
+    if (tlsSkipVerify == true) {
```

Comment:
Is double equals here intentional? I think it allows type conversions before doing the comparison which is a bit ick. My understanding is most projects require triple equals equality checks everywhere for a modicum of type safety.

Reply:
Updated this to also check type for future proofing. 👍

Reply:
Sorry I didn't make it explicit, but the same type-conversion applies to `!=` vs `!==` as well as `==` vs `===`, e.g. see https://stackoverflow.com/a/1946081